app/vue/src/server/config/webpack.config.js

```diff
@@ -52,7 +52,7 @@ export default function() {
       // Based on this CRA feature: https://github.com/facebookincubator/create-react-app/issues/253
       modules: ['node_modules'].concat(nodePaths),
       alias: {
-        'vue$': 'vue/dist/vue.esm.js'
+        'vue': path.resolve(path.join(__dirname, '../../../node_modules', 'vue/dist/vue.esm.js'))
```

Comment:
why do you change it? 🤔
I think `'vue$': 'vue/dist/vue.esm.js'` is correct.

See the docs:
https://vuejs.org/v2/guide/installation.html#Runtime-Compiler-vs-Runtime-only